x/circuit/keeper/msg_authorize_test.go

```diff
@@ -0,0 +1,40 @@
+package keeper
+
+import (
+	"testing"
+
+	"github.com/regen-network/gocuke"
```

Comment:
what is the reasoning to using this fork? what will the maintenance  of this fork look like?

Reply:
It's not a fork. It's a library. Regen will maintain it as needed, which I expect won't be much other than updating to the latest updated gherkin libraries.

Reply:
In the library, the rationale is explained. Basically the existing go cucumber libraries suck, but the official parser is good. This makes it easy to use gherkin with go test